src/lib/addons/feature-event-formatter-md.ts

```diff
@@ -140,6 +142,18 @@ const EVENT_MAP: Record<string, IEventData> = {
         action: '*{{user}}* sent to review change request {{changeRequest}}',
         path: '/projects/{{event.project}}/change-requests/{{event.data.changeRequestId}}',
     },
+    [CHANGE_REQUEST_SCHEDULED]: {
+        action: '*{{user}}* scheduled change request {{changeRequest}} in project *{{event.project}}*',
```

Comment:
Can we add when it was scheduled for? Something like this?
```suggestion
        action: '*{{user}}* scheduled change request {{changeRequest}} to be applied at {{time}} in project *{{event.project}}*',
```